Backend/src/exe/main.cpp

```diff
@@ -0,0 +1,37 @@
+////////////////////////////////////////////////////////////////////////////
+/**
+ *  @file   main.cpp
+ *  @date   Fri August 25 2021
+ *  @brief  This is a basic calculator program which asks for an equation and uses a switch to decifer what the equation is and return the result
+ */
+////////////////////////////////////////////////////////////////////////////
+
+#include "Rest.h"
+
+using namespace calculator;
+
+/**
+ * @brief Main interface which exposes to other components
+ *
+ */
+
+int main() 
+{
+    std::cout << "Starting Server" << std::endl;
+    Pistache::Rest::Router router;
+    calculator::Rest rest; 
+    
+    auto test = Pistache::Rest::Routes::bind(&calculator::Rest::calculateRoute, &rest);
```

Comment:
calculator::Rest::calculateRoute is a little confusing. The route is handled by the router and it binds to 
the "calculate" method.